efficientdet/inference.py

```diff
@@ -491,11 +511,6 @@ def inference(self,
         classes = output_np[:, 6].astype(int)
         scores = output_np[:, 5]
 
-        # This is not needed if disable_pyfun=True
-        # convert [x, y, width, height] to [ymin, xmin, ymax, xmax]
-        # TODO(tanmingxing): make this convertion more efficient.
```

Comment:
You can guard this line with the condition of whether disable_pyfun=True or False.